docs/term_tags/terms.json

```diff
@@ -61,7 +61,7 @@
   },
   "evaluation_parameter_store": {
     "term": "Evaluation Parameter Store",
-    "definition": "A connector to store and retrieve information about parameters used during Validation of an Expectation which reference simple expressions or previously generated metrics.",
+    "definition": "A dynamic value used during Validation of an Expectation which is populated by evaluating simple expressions or by referencing previously generated metrics.",
```

Comment:
I think this is the definition for "Evaluation Parameter" vice "Evaluation Parameter Store"

Reply:
It is.  I've rolled the definition of Evaluation Parameter Store back and changed this to be assigned as the definition of Evaluation Parameter.